SCClassLibrary/SCDoc/SCDocRenderer.sc

```diff
@@ -445,13 +575,13 @@ SCDocHTMLRenderer {
 				};
 				lastargs = args2;
 				case
-					{args2.size>maxargs} {
-						maxargs = args2.size;
-						currentMethod = m2 ?? m;
-					}
-					{args2.size<minArgs} {
-						minArgs = args2.size;
-					};
+				{args2.size>maxargs} {
```

Comment:
Why are you changing whitespaces? Avoid this. Revert to oritinal format. It just confuses the version control.

Reply:
OK, the indentation of these lines was done automatically by pressing the tab key.

Reply:
Never change the indentation. It becomes a wall in the version control, and messes up everything. Be more careful.

Reply:
Check the indentation the source code adopts, and stick to it. Never add or remove whitespaces, because it will become noise and will make version control less efficient.

Reply:
Done!
https://github.com/supercollider/supercollider/pull/6260/commits/be2de2c46486618325e5235b437459284a420c5a